src/views/Form.vue

```diff
@@ -164,6 +165,18 @@ export default {
     }
   },
   methods: {
+    estimateRuntime() {
+      this.runtimeEstimate = Math.round(5 + (this.sourceSchema.size/1000) * (this.targetSchema.size/1000) * 3 / 2000)
```

Comment:
Should this value (the number of seconds estimated) possibly not be saved into the global variable `this.runtimeEstimate`? I get that it's overwritten almost instantaniously, but this might force a re-render with a weird value in the popup, and it's not a value we want shown in the UI. 

I suggest: 
```javascript
    estimateRuntime() {
      const estimateSeconds = Math.round(5 + (this.sourceSchema.size/1000) * (this.targetSchema.size/1000) * 3 / 2000)
      if (estimateSeconds >= 60) {
        this.runtimeEstimate =  estimateSeconds / 60 + " minutes"
      }
      else if (estimateSeconds >= 3600) {
        this.runtimeEstimate = estimateSeconds / 3600  + " hours"
      }
      else {
        this.runtimeEstimate = estimateSeconds  + " seconds"
      }
    },
```